src/controllers/hid/hidcontroller.h

```diff
@@ -40,13 +40,17 @@ class HidController final : public Controller {
 
   private:
     bool isPolling() const override;
+    void processInputReport(int bytesRead);
 
     // For devices which only support a single report, reportID must be set to
     // 0x0.
     void sendBytes(const QByteArray& data) override;
     void sendBytesReport(QByteArray data, unsigned int reportID);
     void sendFeatureReport(const QList<int>& dataList, unsigned int reportID);
 
+    QList<int> getInputReport(unsigned int reportID);
+    QList<int> getFeatureReport(unsigned int reportID);
```

Comment:
Please add brief comments about the use cases for these or links to references about what these mean in the HID protocol.

Reply:
Specifically, the code comment should document how these functions differ from the polling loop which is always running.

Reply:
put the info you explained in https://github.com/mixxxdj/mixxx/pull/3317#issuecomment-770419012 into code comments

Reply:
Done!